builder/virtualbox/ovf/config_test.go

```diff
@@ -30,24 +30,6 @@ func getTempFile(t *testing.T) *os.File {
 	return tf
 }
 
-func testConfigErr(t *testing.T, warns []string, err error) {
```

Comment:
I agree with all the things you say, except I question this one. The reason I removed the helpers was because it makes the tests return the line number of the function call rather than the line where the assertion fails. I removed the helpers to make debugging easier. 

Perhaps there is a golang trick I'm missing here? I'm quite new to the language, so it's possible there was some other value to using the helper functions that I've completely overlooked.